src/collections/Form/Form.js

```diff
@@ -21,13 +22,14 @@ class Form extends Component {
           }
         });
       }
+
       loopThruChildren(props.children);
 
       if (formButtons.length === 0 || !_.include(formButtons, 'submit')) {
         return new Error('`Form` must have `Button` child with type of "submit".');
       }
     },
-    onSubmit: PropTypes.func,
```

Comment:
Nope.  Notice all it did was handle this: `onSubmit={this.props.onSubmit}`.  That means when you submitted the form `this.props.onSubmit` was called with the default params.  Now, that is handled by spreading props as well.  If you add an `onSubmit` prop, it will be added to the component automatically and called automatically in default React style.

Since there a massive amount of props someone might pass down, we are opting to not validate all of them.  Devs should just know, the `onSubmit` prop is a function.  Along with all the other event listeners.  These are regular html concepts.